glyphs2fontir/src/source.rs

```diff
@@ -278,6 +278,7 @@ fn try_name_id(name: &str) -> Option<NameId> {
         "trademarks" => Some(NameId::TRADEMARK),
         "manufacturers" => Some(NameId::MANUFACTURER),
         "designers" => Some(NameId::DESIGNER),
+        "descriptions" => Some(NameId::DESCRIPTION),
```

Comment:
Just want to double check that it's not 'description'?

Reply:
yeah, like the other localizable strings, it's a plural.